src/Pagination/components/PageNumberView.ts

```diff
@@ -41,20 +40,20 @@ export const PageNumberView: SFC<PageNumberViewProps> = (props) => {
             rightButtonNumber = props.pageCount - 1; // last
         }
         // Add first page button
-        pageItems.push(PageNumberButton({ pageNumber: 1, selectedPageNumber, onClickAction, key: "first" }));
+        pageItems.push(createElement(PageNumberButton, { pageNumber: 1, selectedPageNumber, onClickAction: onClickAction.bind(null, 1), key: "first" }));
         if (hasLeftDivider) {
             pageItems.push(createElement(BreakView, { key: "leftDivider" }));
         }
         // Add middle page buttons
         for (let pageNumber = leftButtonNumber; pageNumber <= rightButtonNumber; pageNumber++) {
-            pageItems.push(PageNumberButton({ pageNumber, selectedPageNumber, onClickAction, key: `page${pageNumber}` }));
+            pageItems.push(createElement(PageNumberButton, { pageNumber, selectedPageNumber, onClickAction: onClickAction.bind(null, pageNumber), key: `page${pageNumber}` }));
         }
         if (hasRightDivider) {
             pageItems.push(createElement(BreakView, { key: "rightDivider" }));
         }
         // Add last page button
-        pageItems.push(PageNumberButton({ pageNumber: props.pageCount, selectedPageNumber, onClickAction, key: "last" }));
+        pageItems.push(createElement(PageNumberButton, { pageNumber: props.pageCount, selectedPageNumber, onClickAction: onClickAction.bind(null, props.pageCount), key: "last" }));
     }
 
-    return createElement("ul", { key: props.key }, pageItems);
+    return createElement("ul", {}, pageItems);
```

Comment:
Would be so nice if we could use JSX.. but this widget will be deprecated soon :P

Reply:
Yea, I totally agree